src/index.d.ts

```diff
@@ -98,6 +98,7 @@ export function withTranslation(
 
 export interface I18nextProviderProps {
   i18n: i18n;
+  defaultNS: string;
```

Comment:
This is not a required property, only `i18n` is.  This should be `defaultNS?: string`.  Leaving the test as-is and adding another would prove this true in the future.